schema.cc

```diff
@@ -1704,9 +1704,9 @@ bytes token_column_computation::serialize() const {
     return to_bytes(rjson::print(serialized));
 }
 
-bytes_opt token_column_computation::compute_value(const schema& schema, const partition_key& key, const clustering_row& row) const {
+bytes token_column_computation::compute_value(const schema& schema, const partition_key& key) const {
     auto long_value = dht::token::to_int64(dht::get_token(schema, key));
-    return long_type->decompose(long_value);
+    return {long_type->decompose(long_value)};
```

Comment:
Why the new braces?

Reply:
They are not needed. I removed them now.